grammars/css.cson

```diff
@@ -483,15 +483,15 @@
     ]
   'selector':
     'begin': '\\s*(?=[\\[:.*#a-zA-Z])'
-    'end': '(?=[/@{)])'
+    'end': '(?=[\\/@{\\)])'
```

Comment:
In Oniguruma, the only characters that need to be escaped to be used literally in a character class are `\[]-^`.